content/thirdpartynotices.txt

```diff
@@ -0,0 +1,32 @@
+This file is based on or incorporates material from the projects listed below
+(Third Party IP). The original copyright notice and the license under which
+Microsoft received such Third Party IP, are set forth below. Such licenses and
+notices are provided for informational purposes only. Microsoft licenses the
+Third Party IP to you under the licensing terms for the Microsoft product.
+Microsoft reserves all other rights not expressly granted under this agreement,
+whether by implication, estoppel or otherwise. 
+
+libuv v. 1.7.3
```

Comment:
I took v. 1.7.4 - we will need to rev it once @lodejard change is merged to their repo